src/main/resources/db/changelog/changesets/changelog_20231221T165233Z.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="harperjon (generated)" id="1703177560263-1">
+        <dropColumn columnName="RESULT" tableName="RESULT"/>
```

Comment:
uppercase for H2, lowercase for mysql/postgresql/...
```suggestion
        <dropColumn columnName="result" tableName="result"/>
```

Reply:
we should have added a note in the docs to explain that when removing things, the wrong name (uppercase) is generated and that this is expected in the current version of the system. Or even better if possible fix this problem in a future version and make it generate the correct names with correct casing